media/css/base/banners/vpn-coupon-promo.scss

```diff
@@ -0,0 +1,252 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$brand-theme: 'mozilla';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/button';
+
+.c-coupon-wrapper {
+    background: $color-violet-90;
+    color: $color-white;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+
+    .c-coupon-title,
+    .c-coupon-subtitle,
+    .c-coupon-text {
+        color: $color-white;
+        margin-bottom: 0;
+    }
+
+    .c-coupon-title-wrapper {
+        background-color: $color-violet-50;
+        color: $color-white;
+        padding: $layout-xs;
+        border-bottom-left-radius: 60%;
+        border-bottom-right-radius: 60%;
+        position: relative;
+        height: $spacing-xl;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        .c-coupon-title {
+            @include text-title-sm;
+            text-align: center;
+            padding: $spacing-lg;
+        }
+
+        // White plane icon
+        &::after {
+            position: absolute;
+            content: '';
+            background-image: url('/media/img/banners/vpn/white-plane.svg');
+            background-repeat: no-repeat;
+            background-size: contain;
+            height: 50px;
+            width: 50px;
+            top: 10px;
+            left: 20px;
+        }
+    }
+
+    .c-coupon-content-wrapper {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-around;
+        height: 100%;
+        text-align: center;
+        padding: $spacing-sm;
+        row-gap: $spacing-xl;
+
+        // Class for the banner without the promo code title
+        .c-coupon-title {
+            @include text-title-md;
+        }
+
+        .c-coupon-subtitle {
+            @include text-title-sm;
+        }
+
+        .c-coupon-text {
+            font-weight: bold;
+            @include text-body-md;
+            line-height: 1.8;
+
+            .c-coupon-code {
+                border: $border-radius-xs solid $color-white;
```

Comment:
Just spotted this minor abuse of a token. We've talked about adding tokens for border-width but don't currently have them. This can just be `border: 2px solid $color-white;`